bundles/framework/divmanazer/extension/ExtraFlyout.js

```diff
@@ -98,6 +98,9 @@ Oskari.clazz.define('Oskari.userinterface.extension.ExtraFlyout',
                 } else {
                     me.options.container.append(popup);
                 }
+                if (me.options.showHeading === false) {
+                    popup.find('.oskari-flyoutheading').css('display', 'none');
```

Comment:
This makes a flyout that can't be closed or dragged by normal means other than something that that is inside the content closing it programmatically?

Reply:
Actually this is just a visual change taking off the topmost handle bar above the title row where the closing icon is. The flyout can still be closed by normal means.

Reply:
I think this is not something we need for this. Sure it's a small change but with an overall UI-refresh coming up I'm not comfortable adding more options to current UI-components.

Reply:
Removed the option.